gpkit/small_scripts.py

```diff
@@ -64,16 +64,16 @@ def mag(c):
         return c
 
 
-def unitstr(units, into="%s", options="~"):
+def unitstr(units, into="%s", options="~", dimless='-'):
```

Comment:
Good to have this be settable; is a different string for `dimless` used anywhere?